hazelcast/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheInvalidationTest.java

```diff
@@ -158,6 +169,155 @@ public void testMapClear_shouldClearNearCaches_onOwnerAndBackupNodes() {
         assertNearCacheSizeEventually(clientMap, 0);
     }
 
+
+
```

Comment:
minor: unnecessary new lines, only one new line between the tests is enough

Reply:
Removed